DDDEastAnglia/Helpers/Email/SessionSubmissionEmailSender.cs

```diff
@@ -0,0 +1,67 @@
+﻿using DDDEastAnglia.Helpers.File;
+using DDDEastAnglia.Models;
+using MarkdownSharp;
+using System;
+using System.Net.Mail;
+
+namespace DDDEastAnglia.Helpers.Email
+{
+    public interface ISessionSubmissionEmailSender
+    {
+        void SendEmail(string htmlTemplatePath, string textTemplatePath, Session session, UserProfile speakerProfile, bool updated);
+    }
+
+    public class SessionSubmissionEmailSender : ISessionSubmissionEmailSender
+    {
+        public const string FromEmailAddress = "admin@dddeastanglia.com";
+        public const string FromEmailName = "DDD East Anglia";
+        private const string sessionAbstractToken = "[SessionAbstract]";
+        private const string sessionTitleToken = "[SessionTitle]";
+        private const string votingOpenDateToken = "[VotingOpenDateToken]";
+
+        private readonly IEmailSender _emailSender;
```

Comment:
These shouldn't have underscore prefixes. I think ReSharper's settings' sharing may be failing us :disappointed: